tools/quorumscan/flags/flags.go

```diff
@@ -20,9 +20,10 @@ var (
 		Required: false,
 		EnvVar:   common.PrefixEnvVar(envPrefix, "EIGENDA_DIRECTORY"),
 	}
-	BlsOperatorStateRetrieverFlag = cli.StringFlag{
-		Name:     common.PrefixFlag(FlagPrefix, "bls-operator-state-retriever"),
-		Usage:    "[Deprecated: use EigenDADirectory instead] Address of the BLS operator state Retriever",
+	OperatorStateRetrieverFlag = cli.StringFlag{
+		Name: common.PrefixFlag(FlagPrefix, "bls-operator-state-retriever"),
+		Usage: "[Deprecated: use EigenDADirectory instead] Address of the OperatorStateRetriever contract. " +
+			"Note that the contract no longer uses the BLS prefix.",
 		Required: false,
 		EnvVar:   common.PrefixEnvVar(envPrefix, "BLS_OPERATOR_STATE_RETRIVER"),
 	}
```

Comment:
a bit confused by this comment - iiuc this change is entirely offchain and the deprecated env var is still expressed as `BLS_OPERATOR_STATE_RETRIVER`

Reply:
No the contract is actually called OperatorStateRetriever only now. There is no more bls prefix. check all our deployments on etherscan